### PR TITLE
OCPBUGS-32053: Add examples for some oc adm policy commands

### PR DESCRIPTION
**Pull request by bergerhoffer**
…m-* so they show in the CLI docs

https://issues.redhat.com/browse/OCPBUGS-32053

### CODE REVIEW

**Comment by ardaguclu** [comment]
/retitle OCPBUGS-32053: Add examples for some oc adm policy commands

File: pkg/cli/admin/policy/modify_roles.go [hunk 1]
```diff
@@ -218,6 +238,7 @@ func NewCmdAddClusterRoleToGroup(f kcmdutil.Factory, streams genericiooptions.IO
 		Use:   "add-cluster-role-to-group ROLE GROUP [GROUP...]",
 		Short: "Add a role to groups for all projects in the cluster",
 		Long:  addClusterRoleToGroupLongDesc,
+		Example: addClusterRoleToGroupExample,
```
Comment (ardaguclu) [inline pkg/cli/admin/policy/modify_roles.go:241]: verify failure is real. Because you need to run `make update-gofmt`.

**Comment by ardaguclu** [comment]
/cancel lgtm
no need to retest as verify needs to be fixed.

File: pkg/cli/admin/policy/modify_roles.go (hunk 1 above)
Comment (bergerhoffer) [reply pkg/cli/admin/policy/modify_roles.go:241]: Thanks @ardaguclu! I ran that command, let me know how it looks now

**Pull request merged**